corehq/apps/receiverwrapper/models.py

```diff
@@ -337,6 +353,27 @@ def payload_doc(self, repeat_record):
         return None
 
 
+@register_repeater_type
+class JsonFormRepeater(FormRepeater):
```

Comment:
Seems like that could be incorporated into the payload generator and an additional option to optionally include the app_id.